protocol/0053-PERP-product_builtin_perpetual_future.md

```diff
@@ -1,18 +1,51 @@
-# Built-in [Product](./0051-PROD-product.md): Cash Settled Perpetual Futures (CSF)
+# Built-in [Product](./0051-PROD-product.md): Cash Settled Perpetual Futures
 
-This built-in product provides perpetual futures that are cash-settled, i.e. they are margined and settled in a single asset.
+This built-in product provides perpetual futures contracts that are cash-settled, i.e. they are margined and settled in a single asset, and they never expire.
 
-[Background reading](https://www.paradigm.xyz/2021/05/everlasting-options/#Perpetual_Futures)
+Background reading: [1](https://www.paradigm.xyz/2021/05/everlasting-options/#Perpetual_Futures), [2](https://arxiv.org/pdf/2212.06888.pdf).
 
-Perpetual futures are a simple "delta one" product.
+Perpetual futures are a simple "delta one" product. Mark-to-market settlement occurs with a predefined frequency as per [0003-MTMK-mark_to_market_settlement](0003-MTMK-mark_to_market_settlement.md).  Additionally, a settlement using external data is carried out whenever `settlement_schedule` is triggered. Data obtained from the `settlement_data_cue` and `settlement_data` oracles between to consecutive `settlement_schedule` events is used to calculate the funding rate and exchange cashflows between parties with open positions in the market. A number of protective measures are defined to deal with data availability issues in a predefined way.
 
 ## 1. Product parameters
 
-1. `settlement_data (Data Source: number)`: this data is used by the product to calculate periodic settlement cashflows. The receipt of this data triggers this calculation and the transfers between parties to "true up" to the external reference price.
 1. `settlement_asset (Settlement Asset)`: this is used to specify the single asset that an instrument using this product settles in.
+1. `settlement_schedule (Data Source: datetime)`: this data is used to indicate when the next periodic settlement should be carried out.
+1. `settlement_data_cue (Data Source: datetime)`: this data is used to indicate the earliest time at which the next `settlement_data` should be expected.
```

Comment:
Do we want to make the settlement cue optional and just ingest all the oracle data if it's not specified

Reply:
Optional would work I think.

Reply:
Made it optional, but perhaps it should just be removed altogether now?

Reply:
Removed now.